src/index.ts

```diff
@@ -22,6 +22,7 @@ export {
 } from './auth/credentials';
 export {GCPEnv} from './auth/envDetect';
 export {GoogleAuthOptions, ProjectIdCallback} from './auth/googleauth';
+export {DownscopedClient} from './auth/downscopedclient';
```

Comment:
You shouldn't have exported this yet. This is a feature under construction. Please revert this. This is the last step of the implementation.